wordpress-mobile/WordPress-Aztec-iOS
# Standardize application of formatters

This PR makes the application of all formatters in the textview to follow the same pattern, in order to avoid code redundancy and mistakes.

How to test:
 - Run the app
 - Test the toggle of all formatters in several lines and see how they work

Note: When applying/removing block quotes we are getting some crashes because of the extra new lines, but this was already there. Considering disabling the html application of block quotes in order to avoid crashes on the demo.
